bulk-import/ndjson-generator/NDJsonGenerator.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace NDJsonGenerator
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Text.Json;
+    using Azure.DigitalTwins.Core.Serialization;
+
+    public class NDJsonGenerator
+    {
+        private const string HeaderSection = "{\"Section\": \"Header\"}";
+        private const string ModelsSection = "{\"Section\": \"Models\"}";
+        private const string TwinsSection = "{\"Section\": \"Twins\"}";
+        private const string RelationshipsSection = "{\"Section\": \"Relationships\"}";
+        private const string DefaultHeader = "{\"fileVersion\": \"1.0.0\", \"author\": \"authorName\", \"organization\": \"organization\"}";
+
+        public static Stream GenerateNDJson(IEnumerable<string> models, IEnumerable<BasicDigitalTwin> twins, IEnumerable<BasicRelationship> relationships, string header = "")
+        {
+            var stream = new MemoryStream();
+            var writer = new StreamWriter(stream, Encoding.UTF8);
```

Comment:
Consider wrapping this in a usings statement:

```csharp
using(var writer = new StreamWriter(...))
{
...
}
```

That way, you don't need to worry about disposing of the writer, closing the stream, etc.

Reply:
We dont to dispose the stream yet, it will be later used by consumer to write to some output.